mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -187,6 +192,30 @@ def run(self) -> Tuple[Status, Optional[dict]]:
         _LOG.info("Local run complete: %s ::\n%s", self, data_dict)
         return (Status.SUCCEEDED, data_dict) if data_dict else (Status.FAILED, None)
 
+    def status(self) -> Tuple[Status, List[Tuple[datetime, str, Any]]]:
+
+        (status, _) = super().status()
+        if not (self._is_ready and self._read_telemetry_file):
+            return (status, [])
+
+        assert self._temp_dir is not None
+        try:
+            # FIXME: We should not be assuming that the only output file type is a CSV.
+            data: pandas.DataFrame = pandas.read_csv(
+                self._config_loader_service.resolve_path(
+                    self._read_telemetry_file, extra_paths=[self._temp_dir]))
+        except FileNotFoundError as ex:
+            _LOG.warning("Telemetry CSV file not found: %s :: %s", self._read_telemetry_file, ex)
+            return (status, [])
+
+        _LOG.debug("Read telemetry data:\n%s", data)
+        if list(data.columns) != ["timestamp", "metric", "value"]:
+            _LOG.warning(
+                'Telemetry CSV file should have columns ["timestamp", "metric", "value"] :: %s',
+                self._read_telemetry_file)
```

Comment:
Could also support non-headered CSVs if the columns schema looks to still line up, right?

Reply:
yep, that's why I only print a warning and then proceed with reading the data